unsafe_go121.go

```diff
@@ -0,0 +1,23 @@
+//go:build go1.21
+// +build go1.21
+
+// The unsafe.StringData function was made available in Go 1.20 but it
```

Comment:
I'm confused - if it was added in 1.20, why isn't the build constraint `go1.20`? The go mod directive historically was about enabling language features, not library functions.